README.md

```diff
@@ -22,6 +22,8 @@ documentation](https://optee.readthedocs.io/en/latest/index.html) with the
 addition of `CFG_MS_TPM_20_REF` which must hold the path to a checked out
 copy of the [MS TPM 2.0 Reference
 Implementation](https://github.com/microsoft/ms-tpm-20-ref).
+You need to checkout commit (98b60a44aba79b15fcce1c0d1e46cf5918400f6a) of [MS TPM 2.0 Reference
```

Comment:
I see how this is useful, but it will easily be overlooked when stepping up `ms-tpm-20-ref` in the repo setup.
We need the hash from https://github.com/OP-TEE/manifest/blob/master/common.xml#L20.

Reply:
Sorry, I am not sure how can I read the hash from this common.xml file.

Reply:
Yeah, me neither.